### PR TITLE
series : Added test for issue 14793

### LINKED ISSUE #14793: Limit involving log(factorial(x)) incorrect
From [here](https://mathematica.stackexchange.com/questions/174966/bug-wrong-limit-in-mathematica-11-1-1-and-current-wolframalpha), SymPy also gets this wrong but in a different way:

```
>>> from sympy import *
>>> x = Symbol('x')
>>> e = ((x + S(1)/2) * log(x) - x + log(2*pi)/2 - log(factorial(x)) + S(1)/(12*x))*x**3
>>> limit(e,x,oo)
-oo
>>> N(e.subs(x,1000))
0.00277777698412758
```

The correct answer is `1/360`.

### PR DESCRIPTION
**Pull request by tzublue**
Added test to series/test_series.py
#### References to other Issues or PRs

Fixes #14793

#### Brief description of what is fixed or changed
Added a test in order to close the issue

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/series/tests/test_limits.py [hunk 1]
```diff
@@ -276,6 +276,12 @@ def test_issue_5164():
     assert limit(x**(-0.5), x, 4) == S(4)**(-0.5)
 
 
+def test_issue_14793():
+    expr = ((x + S(1)/2) * log(x) - x + log(2*pi)/2 - \
+        log(factorial(x)) + S(1)/(12*x))*x**3
+    assert simplify(limit(expr, x, oo) - 1/360) == 0
```
Comment (sachin-4099) [inline sympy/series/tests/test_limits.py:282]: This should be `assert limit(expr, x, oo) == S(1)/360`.

Comment (dkodar20) [reply sympy/series/tests/test_limits.py:282]: I see. Are there any particular disadvantage of the way done previously?

Comment (sachin-4099) [reply sympy/series/tests/test_limits.py:282]: The call to `simplify()` function is always costly, it should be avoided as much as possible.

Comment (dkodar20) [reply sympy/series/tests/test_limits.py:282]: Okay, I will make the required changes.

**Comment by sachin-4099** [comment]
Looks good to me. This can be merged @jksuom.

**Comment by jksuom** [comment]
Thanks.

**Pull request merged**